libraries/standard.js

```
@@ -1,8 +1,33 @@
 const lib = new Map();
 
+// check if read values are null, undefined or blank (empty string)
+// dst: destination object, data: data item being processed, 
+// data_field: field being processed, skip_errors: boolean, errors: array of bad data items
+const null_check = (dst, data, data_field, skip_errors, errors) => {
+  // data is null, undefined or an empty object
+  if (!data || Object.keys(data).length === 0) {
+    console.log("No data or data field supplied")
```

I think this can be thrown as an error instead of printing it